CheckDLong: 2.5
Countlist: [1, 2, 3, 4, 5]
DistanceStopLine: 30
PulseMode: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
StatisPeriod: 40
StatisPosition:
  '0': [0]
  '1': [1, 8.0, 0.0, 0.0]
  '2': [1, 2.0, 3.0, 20.0]
  '3': [1, 4.0, 13.0, 13.0]
  '4': [1, 3.0, 5.0, 20.0]
  '5': [1, 23.0, 0.0, 0.0]
  '6': [1, 6.0, 0.0, 0.0]
VehicalLong: [7.0, 3.0, 2.0]
nCount: 5.0
